easybuild/easyconfigs/s/Siesta/Siesta-3.2-pl-5-intel-2016.02-GCC-4.9.eb

```diff
@@ -0,0 +1,65 @@
+easyblock = 'MakeCp'
+
+name = 'Siesta'
+version = '3.2-pl-5'
+
+homepage = 'http://departments.icmab.es/leem/siesta'
+description = """SIESTA is both a method and its computer program implementation, to perform efficient electronic
+ structure calculations and ab initio molecular dynamics simulations of molecules and solids."""
+
+toolchain = {'name': 'intel', 'version': '2016.02-GCC-4.9'}
+toolchainopts = {'usempi': True}
+
+sources = [SOURCELOWER_TGZ]
+source_urls = ['https://dl.dropbox.com/u/20267285/SIESTA-DOWNLOADS/']
+
+# configure is in Src but building in Src is not allowed
+prebuildopts = 'cd Obj && ../Src/obj_setup.sh && ../Src/configure --enable-mpi && '
+
+# build siesta (must override COMP_LIBS in order to ensure usage of MKL's lapack instead of internal)
+buildopts = 'COMP_LIBS="" BLAS_LIBS="$LIBBLAS" LAPACK_LIBS="$LIBLAPACK" BLACS_LIBS="$LIBBLACS" '
+buildopts += 'SCALAPACK_LIBS="$LIBSCALAPACK" && '
```

Comment:
since you're using this in two places, maybe define a local variable for it, and reuse that?

```python
cfg_cmd = '../Src/obj_setup.sh && ../Src/configure --enable-mpi'
# must override COMP_LIBS in order to ensure usage of MKL's lapack instead of internal
build_vars = "COMP_LIBS='' "
build_vars += 'BLAS_LIBS="$LIBBLAS" LAPACK_LIBS="$LIBLAPACK" BLACS_LIBS="$LIBBLACS" SCALAPACK_LIBS="$LIBSCALAPACK"'

# build siesta
prebuildopts = "cd Obj && " + cfg_cmd + ' && '
buildopts = build_vars + ' && '
# build transiesta
buildopts += "cd .. && mkdir Obj2 && cd Obj2 && " + cfg_cmd + " && make transiesta " + build_vars + ' && '
# build all utils
buildopts += 'cd ../Util && sh ./build_all.sh'
```